### PR TITLE
BUG fix: build image will false,if MODULE_PATH is not "dicehub"

### PR DESCRIPTION
**Pull request by ZLValien**
#### What type of this PR

/kind bug

#### What this PR does / why we need it:

image

Added conditional judgment, will not allow wrong code when packaging images

### CODE REVIEW

File: build/dockerfiles/Dockerfile [hunk 1]
```diff
@@ -38,7 +39,7 @@ COPY --from=build "$GOPATH/src/github.com/erda-project/erda/pkg/erda-configs" "/
 COPY --from=build "$GOPATH/src/github.com/erda-project/erda/conf/common" "/app/conf/common"
 
 # use for dicehub
-COPY --from=build "$GOPATH/src/extension" "/app/extension"
+RUN if [ ${MODULE_PATH} = dicehub ] ; then COPY --from=build "$GOPATH/src/extension" "/app/extension" ; fi
```
Comment (sfwn) [inline build/dockerfiles/Dockerfile:42]: amazing

**Review by sfwn — CHANGES_REQUESTED** [review]

File: build/dockerfiles/Dockerfile [hunk 2]
```diff
@@ -8,6 +8,7 @@ WORKDIR "$GOPATH/src/github.com/erda-project/erda/"
 
 ARG MODULE_PATH
 ARG EXTENSION_ZIP_ADDRS
+RUN mkdir -p $GOPATH/src/extension
```
Comment (sfwn) [inline build/dockerfiles/Dockerfile:11]: why extension dir need be located within GOPATH?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: build/dockerfiles/Dockerfile (hunk 2 above)
Comment (ZLValien) [reply build/dockerfiles/Dockerfile]: I remove extension dir to /tmp/dicehub-extension now

**Review by sfwn — APPROVED** [review]

**Pull request merged**